.github/workflows/codeql-analysis.yml

```diff
@@ -1,25 +1,15 @@
 name: "CodeQL"
 
 on:
-  push:   
-    paths-ignore:
-      - 'test/Microsoft.IdentityModel.KeyVaultExtensions.Tests/**'
-      - 'test/Microsoft.IdentityModel.ManagedKeyVaultSecurityKey.Tests/**'
-      - '/src/Microsoft.IdentityModel.KeyVaultExtensions/**'
-      - '/src/Microsoft.IdentityModel.ManagedKeyVaultSecurityKey/**'      
-    branches: [ "dev", "dev6x", "dev7x"]
+  push:       
+    branches: [ "dev" ]
   pull_request:
-    paths-ignore:
-      - 'test/Microsoft.IdentityModel.KeyVaultExtensions.Tests/**'
-      - 'test/Microsoft.IdentityModel.ManagedKeyVaultSecurityKey.Tests/**'
-      - '/src/Microsoft.IdentityModel.KeyVaultExtensions/**'
-      - '/src/Microsoft.IdentityModel.ManagedKeyVaultSecurityKey/**'
     types:
       - opened
       - synchronize
       - reopened
       - ready_for_review
-    branches: [ "dev", "dev6x", "dev7x"]
+    branches: [ "dev" ]
```

Comment:
I think we might want at least dev7 and maybe dev6, they are still active, no?

Reply:
they have codeql on their own respective branches so it will run on PRs into those branches